pkg/cli/fake.go

```diff
@@ -0,0 +1,19 @@
+package cli
```

Comment:
can this be fake_test.go?

Reply:
I think the `_test.go` makes this file a test and not usable as source for another test.
At least, only changing the name breaks the unit test

Reply:
In case I'm missing something, I'll fix this in a following PR